static/sw.js

```diff
@@ -1,8 +1,16 @@
-const CACHE_NAME = 'my-site-cache-v1'; 
+const CACHE_NAME = 'my-site-cache-v1';
 const URLS = [
   '/',
   '/main.js',
-]; 
+];
+const TEAM_NAME = 'jakarta';
+const RATING_FIELD_NAME = 'result';
+const CURSOR = 0;
+const LIMIT = 5;
+const MAX_SCORE = 1000000;
+const DATABASE_NAME = 'gameResult';
+const LEADER_PATH = 'https://ya-praktikum.tech/api/v2/leaderboard'
```

Comment:
❗️
Чтобы константы просовывались сюда, файл воркера нужно собирать в билд тайме. Либо почитать вот это и попробовать:
https://web.dev/es-modules-in-sw/
https://web.dev/module-workers/#what-about-service-worker
```suggestion
const LEADERBORAD_API_ENDPOINT = 'https://ya-praktikum.tech/api/v2/leaderboard'
```